guestbook/README.md

```diff
@@ -4,704 +4,294 @@
-spec:
-  ports:
-    # the port that this service should serve on
-  - port: 6379
-    targetPort: 6379
-  selector:
-    app: redis
-    role: master
-    tier: backend
-```
+### Creating the Redis Master Service
+The guestbook applications needs to communicate to the Redis master to write its data. You need to apply a [Service](https://kubernetes.io/docs/concepts/services-networking/service/) to proxy the traffic to the Redis master Pod. A Service defines a set of Pods and a policy to access them.
 
-[Download example](https://raw.githubusercontent.com/kubernetes/examples/master/guestbook/redis-master-service.yaml)
-<!-- END MUNGE: EXAMPLE redis-master-service.yaml -->
+1. Apply the Redis Master Service from the following `redis-master-service.yaml` file: 
 
-#### Create a Service
+        kubectl apply -f redis-master-service.yaml
 
-According to the [config best practices](http://kubernetes.io/docs/user-guide/config-best-practices/), create a Service before corresponding Deployments so that the scheduler can spread the pods comprising the Service. So we first create the Service by running:
+include code.html language="yaml" file="redis-master-service.yaml" ghlink="/docs/tutorials/docs/tutorials/stateless-application/redis-master-service.yaml"
 
-```console
-$ kubectl create -f guestbook/redis-master-service.yaml
-service "redis-master" created
-```
+**Note:** This manifest file creates a Service named `redis-master` with a set of labels that match the labels previously defined, so the Service routes network traffic to the Redis master Pod.   
+{: .note}
 
-Then check the list of services, which should include the redis-master:
-
-```console
-$ kubectl get services
-NAME              CLUSTER-IP       EXTERNAL-IP       PORT(S)       AGE
-redis-master      10.0.76.248      <none>            6379/TCP      1s
-```
-
-This will cause all pods to see the redis master apparently running on `<CLUSTER-IP>:<PORT>`.  A Service can map an incoming port to any `targetPort` in the backend pod.  Once created, the Service proxy on each node is configured to set up a proxy on the specified port (in this case port `6379`).
-
-`targetPort` will default to `port` if it is omitted in the configuration. `targetPort` is the port the container accepts traffic on, and `port` is the abstracted Service port, which can be any port other pods use to access the Service. For simplicity's sake, we omit it in the following configurations.
-
-The traffic flow from slaves to masters can be described in two steps:
-
-  - A *redis slave* will connect to `port` on the *redis master Service*
-  - Traffic will be forwarded from the Service `port` (on the Service node) to the `targetPort` on the pod that the Service listens to.
-
-For more details, please see [Connecting applications](http://kubernetes.io/docs/user-guide/connecting-applications/).
-
-#### Finding a Service
-
-Kubernetes supports two primary modes of finding a Service — environment variables and DNS.
+{:start="2"}
```

Comment:
there's a chance you might not need the `{:start="2"}` stuff. Consider the following markdown with quotes etc:

Rendering:

-----

1. Hello hello hello
1. Hello this will be a quote:

      > quote quote

1. here's some code yo:

       code code
       i love coding

1. last item (note how I still say 1 and it's still all numbered correctly)

-----

Source:

```
1. Hello hello hello
1. Hello this will be a quote:

      > quote quote

1. here's some code yo:

       code code
       i love coding

1. last item (note how I still say 1 and it's still all numbered correctly)

```


not a big deal, just try it out on staging container (or in a separate PR to docs and see netlify staging). I've got a feeling like we can get rid of jekyll-specific stuff like this and rely more on Markdown semantics. It could be hard to get how many spaces you exactly need to add quotes/code blocks in a list, but it's not impossible.